src/test/java/tatracker/testutil/TypicalTaTracker.java

```diff
@@ -57,4 +57,14 @@ public static TaTracker getTypicalTaTrackerWithGroups() {
         return tat;
     }
 
+    /**
+     * Returns an {@code TaTracker}.
+     */
+    public static TaTracker getTypicalTaTracker() {
+
+        TaTracker tat = new TaTracker();
+
+        return tat;
+    }
+
```

Comment:
Err?

Reply:
HAHA i

- It is needed for when u want an empty TA tracker that u can manipulate urself